src/fsharp/DependencyManager/DependencyManager.fsproj

```diff
@@ -6,7 +6,7 @@
     <OutputType>Library</OutputType>
     <TargetFrameworks>netstandard2.0;net472</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' == 'Unix'">netstandard2.0</TargetFrameworks>
-    <AssemblyName>Interactive.DependencyManager</AssemblyName>
+    <AssemblyName>Microsoft.DotNet.DependencyManager</AssemblyName>
```

Comment:
If we're renaming the project file, let's have it match the final assembly name.

Reply:
@brettfo, I prefer a short meaning full project file name, rather than the information free complete namespace.  May favourite view in github is this:
![image](https://user-images.githubusercontent.com/5175830/80129311-b7bed600-854b-11ea-9adc-2b6d68440254.png)

And my favourite views in vs are these:
![image](https://user-images.githubusercontent.com/5175830/80129425-e472ed80-854b-11ea-8fe0-e1d8fb59a24c.png)

and
![image](https://user-images.githubusercontent.com/5175830/80129521-079d9d00-854c-11ea-9f23-731ba8a3c3aa.png)

I realize that namespace discrimination is important, however, having the most important discrimination at the far right means I have to use up vast screen real estate for the information free bit.

Reply:
The downside to that, of course, is that not having matching artifacts and names makes things extra confusing for anyone who isn't us (and by "us" I also mean us in the future when we forget about these decisions)